LibraBFT/Impl/Consensus/SafetyRules/SafetyRules.agda

```diff
@@ -145,6 +151,65 @@ verifyQcM qc = do
 
 ------------------------------------------------------------------------------
 
+consensusState : SafetyRules → ConsensusState
+consensusState self =
+  ConsensusState∙new (self ^∙ srPersistentStorage ∙ pssSafetyData)
+                     (self ^∙ srPersistentStorage ∙ pssWaypoint)
+                     true -- TODO
+
+------------------------------------------------------------------------------
+
+-- ** NOTE: PAY PARTICULAR ATTENTION TO THIS FUNCTION **
```

Comment:
Please say something about *why* one should pay particular attention to it.  E.g., it is key to understanding how epoch change works, perhaps with a little extra detail about what's important?

Reply:
No fundamental reason.   But I did change the comment to
```
-- ** NOTE: PAY PARTICULAR ATTENTION TO THIS FUNCTION **
-- ** Because : it is long with lots of branches, so easy to transcribe wrong. **
-- ** And if initialization is wrong, everything is wrong. **
```